
Declare four functions that were being called without a visible prototype.

C89 let these through as implicit declarations; clang 16 and later reject them
by default (-Werror,-Wimplicit-function-declaration), so veriT stable2016 does
not build on a current macOS toolchain without this.

All four exist and are exported, they were simply never declared where they are
used.  my_DAG_error and my_DAG_message live in DAG-print.h; byte_to_binary is
defined in unify.c but was declared in no header at all; proof_tmp_eq_norm is
generated by the TMP_PROOF macro in proof.c and is the one member of that family
whose prototype was left out of proof.h.

diff -ru '--exclude=inst-index.c' pristine/veriT-stable2016/src/instantiation/ccfv-constr.c veriT-stable2016/src/instantiation/ccfv-constr.c
--- a/src/instantiation/ccfv-constr.c	2016-10-27 19:55:56.000000000 +0400
+++ b/src/instantiation/ccfv-constr.c	2026-09-09 11:46:31.726639070 +0400
@@ -34,6 +34,7 @@
 #include "pre.h"
 #include "DAG.h"
 #include "DAG-symb.h"
+#include "DAG-print.h"
 
 #include "ccfv-constr.h"
 
diff -ru '--exclude=inst-index.c' pristine/veriT-stable2016/src/instantiation/ccfv-mod.c veriT-stable2016/src/instantiation/ccfv-mod.c
--- a/src/instantiation/ccfv-mod.c	2016-10-27 19:55:56.000000000 +0400
+++ b/src/instantiation/ccfv-mod.c	2026-09-09 11:46:31.726683112 +0400
@@ -33,6 +33,8 @@
 
 #include "DAG-sort.h"
 #include "congruence.h"
+#include "DAG-print.h"
+#include "unify.h"
 
 #include "ccfv-mod.h"
 
diff -ru '--exclude=inst-index.c' pristine/veriT-stable2016/src/instantiation/unify.h veriT-stable2016/src/instantiation/unify.h
--- a/src/instantiation/unify.h	2016-10-27 19:55:56.000000000 +0400
+++ b/src/instantiation/unify.h	2026-09-09 11:46:31.726713476 +0400
@@ -328,6 +328,7 @@
   --------------------------------------------------------------
 */
 
+extern const char *byte_to_binary(unsigned x, unsigned size);
 extern void unify_print(Tunifier unifier);
 extern void print_Tstack_Tunifier(Tstack_Tunifier stack);
 
diff -ru '--exclude=inst-index.c' pristine/veriT-stable2016/src/proof/proof.h veriT-stable2016/src/proof/proof.h
--- a/src/proof/proof.h	2016-10-27 19:55:57.000000000 +0400
+++ b/src/proof/proof.h	2026-09-09 11:47:41.258152371 +0400
@@ -293,6 +293,16 @@
 
 /**
    \author Pascal Fontaine
+   \brief deduce a formula through normalization of equalities between
+   propositions into equivalences
+   \param clause is the original formula
+   \param DAG the result formula
+   \return the id of the proof
+   \remark this is a temporary, high level, proof rule */
+Tproof    proof_tmp_eq_norm(Tproof clause, TDAG DAG);
+
+/**
+   \author Pascal Fontaine
    \brief deduce a formula through elimination of n-ary operators
    \param clause is the original formula
    \param DAG the result formula
